dev-libs/libxslt/libxslt-1.1.34.ebuild

```diff
@@ -19,6 +19,8 @@ KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~m68k ~mips ~ppc ~ppc64 ~riscv ~
 IUSE="crypt debug examples python static-libs elibc_Darwin"
 REQUIRED_USE="python? ( ${PYTHON_REQUIRED_USE} )"
 
+BDEPEND=">=dev-util/pkgconfig-0.29.2"
```

Comment:
No no no no no :)

There's `virtual/pkgconfig` that packages should depend on.

But, more to the point, if a patch claims to fix an error, it should describe what that error is.

Reply:
Oh, but I see you're adding BDEPEND. Okay.

Reply:
I changed it to a BDEPEND on virtual/pkgconfig with a hard blocker on the versions that don't work.

Reply:
And what about pkgconf?